search-relevancy-test/src/search_relevancy_test/reporter.clj

```diff
@@ -20,6 +20,40 @@
                    (ansi/style position :red)))
                index-order))
 
+(defn- result-discounted-cumulative-gain
```

Comment:
We should write unit tests for this function (I know there's probably a few other functions we wrote before that would be good to unit test as well, but this one feels pretty important).

We should also unit test test-discounted-cumulative-gain and reciprocal-rank.